### ISSUE TITLE
Slightly unclear instructions

### ISSUE THREAD

**Issue by DivannKokos** [martymcmodding/iMMERSE#17]
![{3F2ED3B5-3F6F-4E64-A5B1-6B7F1B77E71E}](https://github.com/user-attachments/assets/c99848fa-e446-4013-8629-f96569fea17b)
I presume it supports 6.* reshade, but just to be sure, I need to install latest reshade (which is 6.3.3 at the time of writing), or do I need to install latest 5.* version?

**Comment by DivannKokos** [comment]
If it supports ReShade both 5.* and 6.* then maybe it will be helpful to note it in description as at least version 5.*, if not the latest.

**Comment by martymcmodding** [comment]
Good point, will update the readme. I just never do it because after fixing the 20th typo, the commit history is clogged with BS so...lmao
